git/git_remotes.md

```diff
@@ -1,22 +1,70 @@
 # Git Remotes
-A brief summary about what this lesson is about and why the topics or concepts it covers are important.
+In this lesson, we will cover the `git remote` command and remote repositories. 
+
+A remote is like a bookmark that tells Git where the other versions of a project are stored. These versions are housed in "remote repositories." 
```

Comment:
I'm not sure about the bookmark analogy. I understand the intention. A bookmark is a connection or link to a specific place. Personally I think being explicit that a remote is essentially a link to a version of a project located on another machine would be better here. What do you think?

Reply:
I think that's fine. I struggled a bit with separating a remote from the concept of a remote repository. Being direct will probably be most effective.